source/develop/release-management/features/sla/list-vms-pinned-to-host.html.md

```diff
@@ -41,12 +41,20 @@ Possibilities to expose the new query through the REST API:
   ```
 
 ### Webadmin
-There will be two tables in the `Virtual Machines` sub-tab in the host details page.
-One for running VMs and the other for pinned VMs:
+The table in the `Virtual Machines` sub-tab in the host details page shows
+running VMs and pinned VMs. A radio button on top of it can be used to
+show only one of the groups.
 
-![](/images/wiki/VMs_pinned_to_host.png)
```

Comment:
I think you need to add a 3px top margin to the label to fix the vertical alignment. If you look at the radio button group label in the Disks sub tab of the VMs main tab, you'll see the difference.

Reply:
Good point. Now it looks better.

Reply:
Agreed! =) You get a +1 from me.